doc/ref_model/chapters/chapter02.md

```diff
@@ -266,16 +266,44 @@ By trying to sort workloads into different categories based on the requirements
 <a name="2.4"></a>
 ## 2.4 Cloud Infrastructure Profiles
 
-Based on the above analysis, following cloud infrastructure profiles are proposed (also shown in **Figure 2-1** below)
+Infrastructure profiles are used to tag infrastructure (such as hypervisor hosts, or Kubernetes worker nodes) and associate it with a set of capabilities exploitable by the workloads. *Workload profiles* are requirements expressed as workload metadata, indicating what kind of infrastructure they must run on to achieve functionality and/or the intended level of performance.
+Two profile *layers* are proposed:
+
+- The top level **profiles** represent macro-characteristics that partition infrastructure into separate pools, i.e.: an infrastructure object can belong to one and only one profile, and workloads can only be created using a single profile. Workloads requesting a given profile **must** be instantiated on infrastructure of that same profile.
+- For a given profile, **flavours** represent small deviations from (or further qualification, such as sizing) the profile that do not require partitioning the infrastructure into separate pools, but that have specifications with a finer granularity of the profile. Flavours can be *optionally* requested by workloads that want a more granular control over what infrastructure they can run on, i.e.: an infrastructure resource can have **more than one flavour** attached to it, and workloads can request VMs to be instantiated on infrastructure with a certain flavour. Workloads requesting a given flavour **must** be instantiated on infrastructure with that same flavour. It is allowed to instantiate workloads on infrastructure with more flavours than what is requested, as long as the minimum requirements are satisfied.
```

Comment:
This needs a rewrite. 
Flavours are a resource requesting mechanism. Flavours are composites consisting of such things as sizing information (used to identify infrastructure resources with necessary available capacity) and a set of infrastructure characteristics (such as NUMA alignment, CPU pinning) that are specified by profiles.

It maybe better to state that the workloads request resources specifying some subset of profile characteristics and resource sizing information; workloads specify their resource need or utilise predefined flavours if available (herefater I am using flavour for both types of request). The cloud platform identifies infrastructure resources that can meet the asked for "flavour" specification.

We could state that the workloads specify sizing and a workload profile spec in their resource requests where workload profile specifications are a subset of a node (infrastructure) profile. A resource request can be met by any infrastructure node that has the same or a more specialised profile and the necessary capacity.

In such a write-up we do not have to associate flavours with infrastructure.

Reply:
agreed, have defined node vs workload profiles more clearly at the beginning of 2.4 and added your statements on the mapping between the two, please check https://github.com/cntt-n/CNTT/commit/0ffa1b8d42c44c54cb877dac77ddca4e68062c1a